eahub/base/static/components/profile-edit-image.js

```diff
@@ -1,47 +0,0 @@
-$(document).ready(() => {
```

Comment:
I assume it's tricky to separate the multiselect stuff out from other profile edit actions? If not, it would be nice to move the non-tag related JS (i.e., non multiselect stuff) at a later stage to reduce the scope of this MR.

If that's too difficult, we need a full QA list of all the things we used to be able to do on editing profile to make sure we're not losing any functionality when migrating from vanilla to vue JS on this.

Reply:
I agree, I would be happy to look into it during the profile editing form composition

Reply:
all logic form this file was replaced with django-select2 and native bootstrap5 styles